dataloader.go

```diff
@@ -20,12 +21,17 @@ import (
 // web request.
 type Interface interface {
 	Load(string) Thunk
+	LoadContext(context.Context, string) Thunk
```

Comment:
Thats an interesting naming convention, I went with what the Go team did to the standard lib packages (like sql).